.github/workflows/release.yml

```diff
@@ -0,0 +1,349 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      releaseBranch:
```

Comment:
:question: How is this treated without type information? e.g. option, boolean, etc. Are they all strings? How will it look like in the dispatch form? :thinking:

Reply:
default is string according to https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/

official docs don't specify this well 🤷 
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch